samples/basic/build.gradle

```diff
@@ -33,6 +33,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
 
+    implementation 'com.datadoghq:dd-sdk-android:1.11.0-rc1'
```

Comment:
it should be rebased, `main` already has it (but commented) https://github.com/DataDog/dd-sdk-android-gradle-plugin/blob/4ebeb840a822b5c793fc1847f5cffcf819258f0b/samples/basic/build.gradle#L36